pep-0655.rst

```diff
@@ -55,12 +55,25 @@ a mix of both required and potentially-missing keys:
        title: str
        year: NotRequired[int]
 
+This PEP also makes it possible to define TypedDicts in the
+:pep:`alternative functional syntax <589#alternative-syntax>`
+with a mix of required and potentially-missing keys,
+which is not currently possible at all because the alternative syntax does
+not support inheritance:
+
+::
+
+   Actor = TypedDict('Actor', {
+       'name': str,
+       'in': NotRequired[List[str]],  # "in" is a keyword, so needs alternative syntax
```

Comment:
```suggestion
       # "in" is a keyword, so the functional syntax is necessary
       'in': NotRequired[List[str]],
```

To avoid >79 character line length per PEP 8, you could do this...